src/components/ExpansionSection.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const ExpansionSectionEl = styled.div`
```

Comment:
Hm so this and `Expansion` differ in that one is controlled, one is uncontrolled?

Reply:
According to Product Conventions `Styled components that are not exported should be suffixed with El.` Since I'm exporting a react component, I added `El` here.

Reply:
Oops, I think I put this on the wrong line. I'm not clear what the difference between `ExpansionSection` and `Expansion` are.

Reply:
It takes in the content that is hidden by default and visible only by clicking the `Expansion`. I should rename it to I to `ExpandedContent` and put it as const within `Expansion`..